contracts/FlowIDTableStaking.cdc

```diff
@@ -448,6 +448,14 @@ pub contract FlowIDTableStaking {
 
             // Add the new tokens to tokens committed
             nodeRecord.tokensCommitted.deposit(from: <-tokens)
+
+            // Only add them as a candidate node if they don't already
+            // have tokens staked and are above the minimum
+            if nodeRecord.tokensStaked.balance == 0.0 && FlowIDTableStaking.isGreaterThanMinimumForRole(numTokens: nodeRecord.tokensCommitted.balance, role: nodeRecord.role) {
```

Comment:
This condition is repeated 3 times... I wonder if it would benefit from a helper similar to `isGreaterThanMinimumForRole` that just takes all the values and makes the determination.

This would by DRY it up but make these conditions more descriptive.

Reply:
added in [779898a](https://github.com/onflow/flow-core-contracts/pull/324/commits/779898a7332503673de661051664e30e7fc0950a)